plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseCaseInsensitiveMappingTest.java

```diff
@@ -307,7 +312,7 @@ public void testTableNameClashWithRuleMapping()
     public void testSchemaAndTableNameRuleMapping()
             throws Exception
     {
-        updateRuleBasedIdentifierMappingFile(
+        updateRuleBasedIdentifierMappingFileAndWait(
```

Comment:
Even if you introduce a new method, it could be called `updateRuleBasedIdentifierMappingFile` (without "AndWait")

1. smaller diff
2. it makes it much harder to invoke the old method directly, since it's shadowed

Reply:
Moved sleep to existing method. Thanks.